bind.go

```diff
@@ -179,7 +203,7 @@ func (ml *maybeLineage) checkLensesOrder() error {
 			return err
 		}
 
-		if err := checkLensesOrder(previous, &curr); err != nil {
+		if err := ml.doCheck(previous, &curr, ml.transplan); err != nil {
```

Comment:
Looks like `ml.transplan` is never filled, which I guess it just corresponds to the note left on the description _`and the requisite changes to Translate() aren't yet implemented`_, so just a friendly reminder! :)

Reply:
yeah. `transplan` is actually from the first pass, where i was allowing a mix of CUE and Go. i hadn't cleaned it up yet or firmly decided whether or not to use it.